app/src/main/java/com/example/diceroller/ui/theme/Color.kt

```diff
@@ -17,7 +17,10 @@ package com.example.diceroller.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-val Purple200 = Color(0xFFBB86FC)
-val Purple500 = Color(0xFF6200EE)
-val Purple700 = Color(0xFF3700B3)
-val Teal200 = Color(0xFF03DAC5)
+val Purple80 = Color(0xFFD0BCFF)
```

Comment:
James left the following comment previously but since this in an earlier app may be it is ok.. @MagicalMeghan ?
"Colors: I know the template uses colors + tone number as the identifiers but we're trying to get away from that and name colors after the function they will do so please rename the purple40s and the like to what they are actually doing. In a lot of cases, y'all aren't using the colors anyways so you could comment it out to allow it to default to the baseline scheme."

Reply:
Since we create this app from scratch and aren't talking about color at all I don't know that this makes sense to add into the codelab

Reply:
ack